.github/workflows/docker-gqcp-dev.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy Docker image for GQCP development environment
+
+on:
+  push:
+    paths:
+      - 'Dockerfile'
+    branches:
+      - develop
+  pull_request:
+    paths:
+      - 'Dockerfile'
```

Comment:
To my current Github Actions understanding, this would mean that a PR to develop would overwrite the most current develop-pushed Docker image.

Is there a way we can separate the 'build' of the Docker image (which should be tested `on: pull_request`) from the 'deployment' (which should be done `on: push: develop`)?

(This issue also site...

Reply:
Indeed. Therefore, I have removed the `on: pull_request` build, as I would rather not provide a separate GA Docker debug build. The developers in question should first check the proposed Docker changes on their local/cloud infrastructure and then approve the PR.